pkg/handler/repositories.go

```diff
@@ -684,10 +687,11 @@ func (rh *RepositoryHandler) enqueueAddUploadsEvent(c echo.Context, response api
 			Artifacts:            req.Artifacts,
 			Uploads:              req.Uploads,
 		},
-		OrgId:          orgID,
-		AccountId:      response.AccountID,
-		RepositoryUUID: &response.RepositoryUUID,
-		RequestID:      c.Response().Header().Get(config.HeaderRequestId),
+		OrgId:      orgID,
+		AccountId:  response.AccountID,
+		ObjectUUID: &response.RepositoryUUID,
+		ObjectType: utils.Ptr(config.ObjectTypeRepository),
```

Comment:
could also add the templateUUID and type to the templates.go handler events, like you're doing here

Reply:
I actually did this as part of https://github.com/content-services/content-sources-backend/pull/777/files   the goal here was just to maintain the status quo, but let me know if you want me to do it as part of this PR

Reply:
oh, I missed it there. That's fine then, as long as it's included it somewhere.